src/__tests__/core/block.miner.test.ts

```diff
@@ -18,7 +23,12 @@ test('block->miner: successfully returns miner account', async () => {
       block: {
         miner: {
           address: '0x829BD824B016326A401d083B33D092293333A830',
-          code: null,
+          balance: 5.112102994617621e21,
```

Comment:
Since the balance can change, this test can fail unexpectedly. So instead of requesting the balance, you can request the `type` (`EXTERNALLY_OWNED`) to verify that the fields under the `Account` are indeed resolvable.